freud/locality.pyx

```diff
@@ -1099,6 +1099,9 @@ cdef class PeriodicBuffer(_Compute):
                 each side, meaning that one image doubles the box side lengths,
                 two images triples the box side lengths, and so on.
                 (Default value = :code:`False`).
+            include_origin (bool, optional):
```

Comment:
I'd like this argument's name to  be a little different. Some options might be `include_origin_image` or `include_input_points`? I'd welcome some back and forth on this.